src/main/java/bio/terra/testrunner/runner/config/KubernetesSpecification.java

```diff
@@ -1,13 +1,16 @@
 package bio.terra.testrunner.runner.config;
 
+import org.graalvm.compiler.core.common.SuppressFBWarnings;
+
 public class KubernetesSpecification implements SpecificationInterface {
-  public int numberOfInitialPods = 1;
+  @SuppressFBWarnings(value = "UWF_NULL_FIELD", justification = "")
```

Comment:
Please include the justification message.

Reply:
Justification added: `Forcing numberOfInitialPods to accept null as a valid scenario`